diff --git a/build/make/configure.sh b/build/make/configure.sh
index c99a01c..90e3cc3 100755
--- a/build/make/configure.sh
+++ b/build/make/configure.sh
@@ -482,7 +482,8 @@ process_common_cmdline() {
             logfile=/dev/null
         fi
         ;;
-        --target=*) toolchain="${toolchain:-${optval}}"
+        --target=*) 
+	# toolchain="${toolchain:-${optval}}"
         ;;
         --force-target=*) toolchain="${toolchain:-${optval}}"; enable force_toolchain
         ;;
@@ -532,7 +533,7 @@ process_common_cmdline() {
         prefix="${optval}"
         ;;
         --libdir=*)
-        libdir="${optval}"
+        #libdir="${optval}"
         ;;
         --sdk-path=*)
         [ -d "${optval}" ] || die "Not a directory: ${optval}"
@@ -543,6 +544,34 @@ process_common_cmdline() {
         ;;
         --help|-h) show_help
         ;;
+	--build=*)
+	;;
+	--host=*)
+	;;
+	--program-prefix=*)
+	;;
+	--exec-prefix=*)
+	;;
+	--bindir=*)
+	;;
+	--sbindir=*)
+	;;
+	--sysconfdir=*)
+	;;
+	--datadir=*)
+	;;
+	--includedir=*)
+	;;
+	--libexecdir=*)
+	;;
+	--localstatedir=*)
+	;;
+	--sharedstatedir=*)
+	;;
+	--mandir=*)
+	;;
+	--infodir=*)
+	;;
         *) die_unknown $opt
         ;;
         esac
